src/Pact/Types/Runtime.hs

```diff
@@ -232,15 +234,18 @@ toPactId = PactId . hashToText
 -- | Dynamic storage for loaded names and modules, and current namespace.
 data RefState = RefState {
       -- | Imported Module-local defs and natives.
-      _rsLoaded :: HM.HashMap Name Ref
+      _rsLoaded :: HM.HashMap Text (Ref, Maybe (ModuleHash))
```

Comment:
the `Maybe` is just for repl natives right? Can we improve on this somehow? ie in Repl, use `local` to elide repl natives when loading a `.pact` file?

Reply:
Interface consts and schemas have that set as `Nothing` as well.